### DISCUSSION TITLE
Solito + Next.js Layouts

### DISCUSSION THREAD

**Discussion by acrabb** [nandorojo/solito#161] [General]
Hey team, what're the thoughts on supporting Next.js's Layout change coming later this year?
What would be required to support it?

https://nextjs.org/blog/layouts-rfc

**Comment by nandorojo** [comment]
I'm not sure. Depends on which pieces. On the native side, you'd have to decide if you want to reuse the same `Layout` component. In many cases, you won't want to. This is a potentially good place for platform-specific code, such as headers and sidebars.

I'm banking on the upcoming Expo file system router, though. It is taking inspiration from the new Next.js changes, among others.
